Description: Allow override of pkg-config location to
 support cross-architecture builds.
Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=960512
Author: Helmut Grohne <helmut@subdivi.de>
--- a/Makefile
+++ b/Makefile
@@ -14,11 +14,13 @@
 		kbfunc.o strlcpy.o strlcat.o y.tab.o \
 		strtonum.o reallocarray.o
 
-CPPFLAGS+=	`pkg-config --cflags x11 xft xrandr`
+PKG_CONFIG?=	pkg-config
+
+CPPFLAGS+=	`$(PKG_CONFIG) --cflags x11 xft xrandr`
 
 CFLAGS+=	-Wall -O2 -g -D_GNU_SOURCE
 
-LDFLAGS+=	`pkg-config --libs x11 xft xrandr`
+LDFLAGS+=	`$(PKG_CONFIG) --libs x11 xft xrandr`
 
 MANPREFIX?=	${PREFIX}/share/man
 
